packages/flutter_tools/bin/linux_backend.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
```

Comment:
Shouldn't we have a `set -e` in this script? Right now copy failures wouldn't fail the script.